Sources/ProjectDescription/Template/Template.swift

```diff
@@ -48,7 +48,25 @@ public struct Template: Codable, Equatable {
         /// Required attribute with a given name
         case required(String)
         /// Optional attribute with a given name and a default value used when attribute not provided by user
-        case optional(String, default: String)
+        case optional(String, default: Value)
+    }
+}
+
+extension Template.Attribute {
+    /// This represents the default value type of Attribute
+    public indirect enum Value: Codable, Equatable {
```

Comment:
You should add all the `ExpressibleByXXXLiteral` conformances, so you can pass literals directly (e.g. `"ciao"` instead of `.string("ciao")`).

Reply:
Thank you for the guidance